src/main/java/xyz/nucleoid/fantasy/util/VoidChunkGenerator.java

```diff
@@ -104,10 +105,20 @@ public VoidChunkGenerator(Registry<Biome> biomeRegistry, RegistryKey<Biome> biom
         this(biomeRegistry.getEntry(biome).get());
     }
 
+    // Create an empty (void) world!
+    public VoidChunkGenerator(MinecraftServer server) {
+        this(RegistryEntry.of(server.getRegistryManager().get(RegistryKeys.BIOME).getEntry(0).get()));
+    }
+    // Create a world with a given Biome (as an ID)
+    public VoidChunkGenerator(MinecraftServer server, Identifier biome) {
+        this(RegistryEntry.of(server.getRegistryManager().get(RegistryKeys.BIOME).get(biome)));
```

Comment:
This assumes the developer has properly registered their biome. Should RegistryKey<Biome> be used instead of an Identifier?